From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Mon, 10 Jul 2023 22:05:08 +0200
Subject: [PATCH] Material fixes - Fix LEGACY_ELYTRA durability

Corrects the `Material.getMaxDurability` value of `Material.LEGACY_ELYTRA`.

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Fiddle - https://fiddlemc.org

diff --git a/src/main/java/org/bukkit/Material.java b/src/main/java/org/bukkit/Material.java
index ff6da47034b7cd44f59b8b50a92bda78b387cf04..eb9cd9a8a0b5b0f249e62cddf4cc37e4ac5f38c6 100644
--- a/src/main/java/org/bukkit/Material.java
+++ b/src/main/java/org/bukkit/Material.java
@@ -4337,7 +4337,7 @@ public enum Material implements Keyed, Translatable, net.kyori.adventure.transla
     @Deprecated
     LEGACY_SHIELD(442, 1, 336),
     @Deprecated
-    LEGACY_ELYTRA(443, 1, 431),
+    LEGACY_ELYTRA(443, 1, 432), // Fiddle - correct legacy elytra durability
     @Deprecated
     LEGACY_BOAT_SPRUCE(444, 1),
     @Deprecated
